app/Core/ALPSVersions.php

```diff
@@ -55,6 +55,11 @@ public static function get()
         return self::getFallbackVersion();
     }
 
+    public static function getLocalCachedVersion() {
+        $cachedVersion = scandir(get_stylesheet_directory().self::LOCAL_PATH)[2];
+        return $cachedVersion ? $cachedVersion : 'Local styles was not cached yet!';
```

Comment:
Change `Local styles was not cached yet!` --> `Local styles are not cached yet!`

Reply:
Thank you. It was fixed